library/src/main/java/com/alamkanak/weekview/WeekView.java

```diff
@@ -2488,6 +2528,10 @@ private boolean forceFinishScroll() {
         }
     }
 
+    public void setCustomScrolableDaysEnabled(boolean isEnabled){
```

Comment:
The name if this setter has to change, according to the new name for the field.

Reply:
Ok